Svelte-check returns error "Cannot find module" on image import when using TypeScript

`svelte-check` reports an error when using TypeScript on the image imports used in the Sapper template project:

```
sapper-project/src/routes/index.svelte:2:25
Error: Cannot find module 'images/successkid.jpg' or its corresponding type declarations. (ts)
<script lang='ts'>
	import successkid from 'images/successkid.jpg';
</script>
```

The images are located in `src/node_modules/images` (since sveltejs/sapper-template#248). This is closely related to sveltejs/language-tools#610 but in this case it is `svelte-check` that reports an error whereas for sveltejs/language-tools#610 the error only displays in VSCode, so I suspect it's two different issues.

**To Reproduce**
```
npx degit "sveltejs/sapper-template#rollup" sapper-project
cd sapper-project
node scripts/setupTypeScript.js
yarn
yarn run build
yarn run validate
```

Ping @benmccann 

**Versions**
Svelte for VSCode v102.2.0
[sapper template](https://github.com/sveltejs/sapper-template-rollup/commit/2d3df9051bb40fcaeb838b0a7566bc532462d5e3)
Sapper 0.28.10
Svelte 3.29.0
Svelte-check 1.0.59

--- a/src/ambient.d.ts
+++ b/src/ambient.d.ts
@@ -8,12 +8,32 @@
 		<img src="{successkid}">
 	 ```
  */
+declare module "*.gif" {
+	const value: string;
+	export = value;
+}
+
 declare module "*.jpg" {
-	const value: any;
+	const value: string;
+	export = value;
+}
+
+declare module "*.jpeg" {
+	const value: string;
 	export = value;
 }
 
 declare module "*.png" {
-	const value: any;
+	const value: string;
+	export = value;
+}
+
+declare module "*.svg" {
+	const value: string;
+	export = value;
+}
+
+declare module "*.webp" {
+	const value: string;
 	export = value;
 }

--- a/src/ambient.d.ts
+++ b/src/ambient.d.ts
@@ -1,0 +1,19 @@
+/**
+ * These declarations tell TypeScript that we allow import of images, e.g.
+ * ```
+		<script lang='ts'>
+			import successkid from 'images/successkid.jpg';
+		</script>
+
+		<img src="{successkid}">
+	 ```
+ */
+declare module "*.jpg" {
+	const value: any;
+	export = value;
+}
+
+declare module "*.png" {
+	const value: any;
+	export = value;
+}

